lib/libyui_client/widgets/numberbox.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module LibyuiClient
+  module Widgets
+    # Class representing a numberbox in the UI. It can be YIntField.
+    class Numberbox < Widgets::Base
+      # Returns minimum value to set in the numberbox
+      # @return [Integer] with minimum value
+      # @example Get minimum value to set numberbox with id 'test'
+      #   {
+      #      "class": "YIntField",
+      #      "debug_label": "label_test",
+      #      "hstretch": true,
+      #      "id": "test",
+      #      "label": "label_test",
+      #      "max_value": 65535,
+      #      "min_value": 0,
+      #      "value": 3260
+      #   }
+      #   app.numberbox(id: 'test').min_value
+      def min_value
+        property(:min_value)
+      end
+
+      # Returns maximum value to set in the numberbox
+      # @return [Integer] with maximum value
+      # @example Get maximum value to set numberbox with id 'test'
+      #   {
+      #      "class": "YIntField",
+      #      "debug_label": "label_test",
+      #      "hstretch": true,
+      #      "id": "test",
+      #      "label": "label_test",
+      #      "max_value": 65535,
+      #      "min_value": 0,
+      #      "value": 3260
+      #   }
+      #   app.numberbox(id: 'test').max_value
+      def max_value
+        property(:max_value)
+      end
+
+      # Sends action to set the value of numberbox.
+      # @param value [String] text to be set in numberbox
+      # @return [Numberbox] in case action is successful
+      # @example Set text in numberbox with id 'test' to 'my value'
+      #   app.numberbox(id: 'test').set('my value')
```

Comment:
Is it possible to send the String value there? I supposed Numberbox is allowing to enter only numbers. Could you please update the doc to `.set(123)` in case only numbers are allowed. Just to avoid confusion.

Reply:
fixed